custom_components/tahoma/climate_devices/atlantic_heat_recovery_ventilation.py

```diff
@@ -0,0 +1,247 @@
+"""Support for AtlanticHeatRecoveryVentilation."""
+import logging
+from typing import List, Optional
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    FAN_AUTO,
+    HVAC_MODE_FAN_ONLY,
+    SUPPORT_FAN_MODE,
+    SUPPORT_PRESET_MODE,
+)
+from homeassistant.const import EVENT_HOMEASSISTANT_START, STATE_UNKNOWN, TEMP_CELSIUS
+from homeassistant.core import callback
+from homeassistant.helpers.event import async_track_state_change
+
+from ..coordinator import TahomaDataUpdateCoordinator
+from ..tahoma_entity import TahomaEntity
+
+FAN_BOOST = "home_boost"
+FAN_KITCHEN = "kitchen_boost"
+FAN_AWAY = "away"
+FAN_BYPASS = "bypass_boost"
+
+PRESET_AUTO = "auto"
+PRESET_PROG = "prog"
+PRESET_MANUAL = "manual"
+
+COMMAND_SET_AIR_DEMAND_MODE = "setAirDemandMode"
+COMMAND_SET_VENTILATION_CONFIGURATION_MODE = "setVentilationConfigurationMode"
+COMMAND_SET_VENTILATION_MODE = "setVentilationMode"
+COMMAND_REFRESH_VENTILATION_STATE = "refreshVentilationState"
+COMMAND_REFRESH_VENTILATION_CONFIGURATION_MODE = "refreshVentilationConfigurationMode"
+
+IO_AIR_DEMAND_MODE_STATE = "io:AirDemandModeState"
+IO_VENTILATION_MODE_STATE = "io:VentilationModeState"
+IO_VENTILATION_CONFIGURATION_MODE_STATE = "io:VentilationConfigurationModeState"
```

Comment:
Can we use pyoverkiz enums for all these constants?

Reply:
Done